riscv/interactive.cc

```diff
@@ -110,7 +112,8 @@ std::string sim_t::rin(streambuf *bout_ptr) {
 
 // write sout to socket (via bout)
 void sim_t::wout(streambuf *bout_ptr) {
-  if (acceptor_ptr) { // only if a socket has been created
+  if (cmd_file==NULL && acceptor_ptr) { // only if  we are not getting command inputs from a file
```

Comment:
spaces around `==` operator. Also, it seems a little weird to use an explicit comparison to NULL for the first part and an implicit comparison in the second half, i.e., either `!cmd_file && acceptor_ptr` or `cmd_file == NULL && acceptor_ptr != NULL` is fine, but avoid mixing and matching styles.